src/uct/ib/base/ib_device.c

```diff
@@ -105,7 +105,25 @@ static uct_ib_device_spec_t uct_ib_builtin_device_specs[] = {
    UCT_IB_DEVICE_FLAG_DC_V2, 50},
   {"ConnectX-6 DX", {0x15b3, 4125},
    UCT_IB_DEVICE_FLAG_MELLANOX | UCT_IB_DEVICE_FLAG_MLX5_PRM |
-   UCT_IB_DEVICE_FLAG_DC_V2, 50},
+   UCT_IB_DEVICE_FLAG_DC_V2, 60},
+  {"ConnectX-6 DX VF", {0x15b3, 4126},
+   UCT_IB_DEVICE_FLAG_MELLANOX | UCT_IB_DEVICE_FLAG_MLX5_PRM |
+   UCT_IB_DEVICE_FLAG_DC_V2, 60},
+  {"ConnectX-6 LX", {0x15b3, 4127},
+   UCT_IB_DEVICE_FLAG_MELLANOX | UCT_IB_DEVICE_FLAG_MLX5_PRM |
+   UCT_IB_DEVICE_FLAG_DC_V2, 45},
+  {"ConnectX-7", {0x15b3, 4129},
+   UCT_IB_DEVICE_FLAG_MELLANOX | UCT_IB_DEVICE_FLAG_MLX5_PRM |
+   UCT_IB_DEVICE_FLAG_DC_V2, 70},
+  {"BlueField", {0x15b3, 0xa2d2},
+   UCT_IB_DEVICE_FLAG_MELLANOX | UCT_IB_DEVICE_FLAG_MLX5_PRM |
```

Comment:
are we going to expose any additional features for BlueFields?

Reply:
not at the moment